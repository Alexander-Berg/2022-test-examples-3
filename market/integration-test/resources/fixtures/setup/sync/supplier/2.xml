<?xml version="1.0" encoding="UTF-8"?>
<dataset>
    <system_property
        property_name = "ENABLE_YT_SUPPLIER_SYNC"
        property_value = "true"
    />
    <supplier
        ID="552977"
        NAME="Hit-cosmetic"
        DROPSHIP="false"
        FULFILLMENT="true"
        CROSSDOCK="false"
        CLICK_AND_COLLECT="false"
        DSBS="false"
        CPC="false"
    />
</dataset>
