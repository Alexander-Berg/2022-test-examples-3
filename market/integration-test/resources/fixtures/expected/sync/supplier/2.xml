<?xml version="1.0" encoding="UTF-8"?>
<dataset>
    <supplier
        ID="10537861"
        NAME="DROPSHIP"
        DROPSHIP="true"
        FULFILLMENT="false"
        CROSSDOCK="false"
        CLICK_AND_COLLECT="false"
        DSBS="false"
        CPC="false"
    />

    <supplier
        ID="10537913"
        NAME="CROSSDOCK"
        DROPSHIP="false"
        FULFILLMENT="true"
        CROSSDOCK="true"
        CLICK_AND_COLLECT="false"
        DSBS="false"
        CPC="false"
    />
    <supplier
        ID="552977"
        NAME="Hit-cosmetic"
        DROPSHIP="false"
        FULFILLMENT="true"
        CROSSDOCK="true"
        CLICK_AND_COLLECT="false"
        DSBS="false"
        CPC="false"
    />
</dataset>
